use_case_guide/multi_tenant.rst

```diff
@@ -0,0 +1,517 @@
+    clicked_at timestamp without time zone NOT NULL,
+    site_url text NOT NULL,
+    cost_per_click_usd numeric(20,10),
+    user_ip inet NOT NULL,
+    user_data jsonb NOT NULL
+  );
+
+  CREATE TABLE impressions (
+    id bigserial PRIMARY KEY,
+    ad_id bigint REFERENCES ads (id),
+    seen_at timestamp without time zone NOT NULL,
+    site_url text NOT NULL,
+    cost_per_impression_usd numeric(20,10),
+    user_ip inet NOT NULL,
+    user_data jsonb NOT NULL
+  );
+
+There are modifications we can make to the schema which will give it a performance boost in a distributed environment like Citus. To see how, we must become familiar with how Citus distributes data and executes queries.
+
+Scaling the Relational Data Model
+---------------------------------
+
+The relational data model is great for applications. It protects data integrity, allows flexible queries, and accommodates changing data. Traditionally the only problem was that relational databases weren't considered capable of scaling to the workloads needed for big SaaS applications. Developers had to put up with awkward NoSQL databases -- or a collection of backend services -- to reach that size. With Citus you can keep your data model *and* make it scale.
+
+Citus appears to applications as a single PostgreSQL database, but it internally routes queries to an adjustable number of physical servers (nodes) which can process requests in parallel. When configuring Citus, the database administrator specifies how the routing will happen.
+
+Multi-tenant applications have a nice property that we can take advantage of: queries usually always request information for one tenant at a time, not a mix of tenants. For instance, when a salesperson is searching prospect information in a CRM, the search results are specific to his employer; other businesses' leads and notes are not included.
+
+Because application queries are restricted to a single tenant, such as a store or company, one approach for making multi-tenant application queries fast is to store *all* data for a given tenant on the same node. This minimizes network overhead between the nodes and allows Citus to support all your application's joins, key constraints and transactions efficiently. They can happen entirely within a node and do not require information from other nodes. With this, you can scale across multiple nodes without having to totally re-write or re-architect your application.
+
+We do this in Citus by making sure every table in our schema has a column to clearly mark which tenant owns which rows. In the ad analytics application the tenants are companies, so we must ensure all tables have a :code:`company_id` column.
+
+We can tell Citus to use this column to read and write rows to the same node when the rows are marked for the same company. In Citus' terminology :code:`company_id` will be the *distribution column*, which you can learn more about in :ref:`Distributed Data Modeling <distributed_data_modeling>`.
```

Comment:
Could we link here to distributing by tenant instead of the general distributed data modeling? (https://citus-doc-preview-pr-365.herokuapp.com/sharding/data_modeling.html#distributing-by-tenant-id)